Translation corrections (#466)
07th-mod/higurashi-console-arcs: Update/omot_016.txt

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#a59da9>大石</color>", NULL, "<color=#a59da9>Ooishi</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 11, "ps3/s14/11/210700094", 256, TRUE);
	OutputLine(NULL, "　しかも麻薬取締法違反って、令状もないのにウソ言っちゃいけませんよ～！」",

> 			NULL, " And what's more, you don't have a warrant, so you shouldn't be tellings lies about violating the Narcotics Control Act~!\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " And what's more, you don't even have a warrant! You can't be telling lies about violating the Narcotics Control Act~!\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#797d8a>赤坂</color>", NULL, "<color=#797d8a>Akasaka</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 13, "ps3/s14/13/211300099", 256, TRUE);
	OutputLine(NULL, "「調書でつじつまを合わせれば、何とでもなりますよ。",

> 			NULL, "\"You can do anything as long as it's consistent with the police report.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"You can do anything as long as it adds up in the police report.", Line_WaitForInput);
```
